src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -23,6 +23,8 @@ class ExecutableFinder
 
     /**
      * Replaces default suffixes of executable.
+     *
+     * @param array $suffixes
```

Comment:
This can be removed as all the information are already part of the method signature.